src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -185,12 +185,14 @@ public async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAccount(st
                     OrgainsationCount = accountStatsResponse?.Stats?.OrganisationCount ?? 0,
                     PayeSchemeCount = accountStatsResponse?.Stats?.PayeSchemeCount ?? 0,
                     TeamMemberCount = accountStatsResponse?.Stats?.TeamMemberCount ?? 0,
+                    TeamMembersInvited = accountStatsResponse?.Stats?.TeamMembersInvited ?? 0,
                     ShowWizard = showWizard,
                     ShowAcademicYearBanner = _currentDateTime.Now < new DateTime(2017, 10, 20),
                     Tasks = tasks,
                     HashedAccountId = accountId,
                     RequiresAgreementSigning = requiresAgreementSigning,
-                    AgreementsToSign = requiresAgreementSigning > 0
+                    AgreementsToSign = requiresAgreementSigning > 0,
+                    SignedAgreementCount = accountStatsResponse?.Stats?.SignedAgreementCount ?? 0
```

Comment:
rather than pulling this from stats as an extra query we already have an EmployerAgreementsResponse in the code above which we could get a count of the signed agreements from

Reply:
Agree with this and has been rectified